doc/sphinx/src/building.rst

```diff
@@ -108,6 +108,7 @@ The main CMake options to configure building are in the following table:
  ``SINGULARITY_BUILD_CLOSURE``           OFF      Build the mixed cell closure models
  ``SINGULARITY_BUILD_TESTS``             OFF      Build test infrastructure.
  ``SINGULARITY_BUILD_PYTHON``            OFF      Build Python bindings.
+ ``SINGULARITY_BUILD_PYTHON``            OFF      Build examples of ``singularity-eos`` in use.
```

Comment:
`SINGULARITY_BUILD_EXAMPLES` not `SINGULARITY_BUILD_PYTHON`

Reply:
Oops! Good catch!